### PR TITLE
Admin: fix issues and properly test order related querysets

### PR DESCRIPTION
**Pull request by tulimaki**

### CODE REVIEW

File: shuup/admin/modules/orders/views/addresses.py [hunk 1]
```diff
@@ -26,6 +26,10 @@ class OrderAddressEditView(UpdateView):
     context_object_name = "order"
     template_name = "shuup/admin/orders/_address_edit.jinja"
 
+    def get_queryset(self):
+        shop_ids = Shop.objects.get_for_user(self.request.user).values_list("id", flat=True)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/addresses.py]: maybe use get_shop here

File: shuup/admin/modules/orders/views/detail.py [hunk 2]
```diff
@@ -30,7 +30,8 @@ def get_toolbar(self):
         return OrderDetailToolbar(self.object)
 
     def get_queryset(self):
-        return super(OrderDetailView, self).get_queryset().exclude(deleted=True).filter(shop=get_shop(self.request))
+        shop_ids = Shop.objects.get_for_user(self.request.user).values_list("id", flat=True)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/detail.py]: get_shop()

File: shuup/admin/modules/orders/views/detail.py [hunk 3]
```diff
@@ -53,6 +54,10 @@ def get_context_data(self, **kwargs):
 class OrderSetStatusView(DetailView):
     model = Order
 
+    def get_queryset(self):
+        shop_ids = Shop.objects.get_for_user(self.request.user).values_list("id", flat=True)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/detail.py]: get_shop()

File: shuup/admin/modules/orders/views/addresses.py [hunk 4]
```diff
@@ -26,6 +26,10 @@ class OrderAddressEditView(UpdateView):
...
+        return Order.objects.exclude(deleted=True).filter(shop_id__in=shop_ids)
```
Comment (chessbr) [inline shuup/admin/modules/orders/views/addresses.py]: why not just
```py
Order.objects.exclude(deleted=True).filter(shop_in=Shop.objects.get_for_user(get_shop(self.request)))
```

Comment (tulimaki) [reply shuup/admin/modules/orders/views/addresses.py]: get_for_user does not take shop as param :)

File: shuup/admin/modules/orders/views/addresses.py (hunk 1 above)
Comment (tulimaki) [reply shuup/admin/modules/orders/views/addresses.py]: For these I do not see necessary for current shop to be active. I just want to ensure that not wrong people can access these views.

**Pull request merged**